package.json

```diff
@@ -78,6 +79,7 @@
     "phantomjs-prebuilt": "^2.1.14",
     "react": "^15.x",
     "react-dom": "^15.x",
+    "react-frame-component": "^1.0.3",
```

Comment:
It looks like this is only being used in tests, can you move this to `devDependencies`?